core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1164,7 +1174,10 @@ class WorkspaceService(protected val ctx: RawlsRequestContext,
                 _ <- methodConfig.traverse_(dataAccess.methodConfigurationQuery.create(destWorkspaceContext, _))
               } yield ()
             })
-
+            _ <- traceDBIOWithParent("FastPassService.setupFastPassClonedWorkspace", parentContext)(childContext =>
```

Comment:
Do we need to worry about the comment at line 1100-- "Orchestration has its own implementation of cloneWorkspace. When changing something here, you may also need to update orchestration's implementation"?

Reply:
It looks like orch just calls out to the clone workspace endpoint on rawls, since we arent changing any endpoint behavior we should be fine.

Reply:
Thanks for checking. Perhaps we should consider changing the comment in the future….